metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -427,6 +428,10 @@ std::vector<Alignment> DefaultColumnExtender::extend(score_t min_path_score,
     table.clear();
     prev_starts.clear();
 
+    // prevent overflow
+    if (config_.xdrop > std::numeric_limits<score_t>::max() - added_xdrop)
+        added_xdrop = std::numeric_limits<score_t>::max() - config_.xdrop;
```

Comment:
Rather than changing the value, wouldn't it be better to do the checks below more carefully so you don't let the overflow happen?

Reply:
The point is that I want the xdrop parameter to be the saturated addition of `config_.xdrop` and `added_xdrop`. So another way I could do it is to just define a new variable as such called `xdrop` and use that everywhere instead.

Reply:
This allows the user to disable the xdrop heuristic by setting the xdrop cutoff to max_int.